compiler-plugin-tests/src/test/resources/ballerina_sources/invalid_service_11/Ballerina.toml

```
@@ -1,4 +1,4 @@
 [package]
 org = "ftp_test"
-name = "invalid_service_11"
+name = "invalid_service_17"
```

```suggestion
name = "invalid_service_11"
```
This is not related to java version

---

Yes, it's due to a mistake while replacing